lib/outh/google-outh.ts

```diff
@@ -0,0 +1,102 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
I think this file will help us to authorize to gdrive for `timeline-viewer` integration.